### PR TITLE
README Amendments

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Each commit is nice and atomic (with exception of fixing the trailing spaces in blank lines, pesky VSCode!), so should be easy to review.

## Improvements 
- Not all volunteers are at the University of Southampton
- Linking to things is useful for attribution etc (the [second amendment](https://img07.deviantart.net/237a/i/2015/082/3/0/beararms_2_by_uhd4k-d8mtvu6.png), if you will)
- Improve wording of kit support to remove _subjectively_ misleading footnote about official support.

I'm not entirely happy with the wording of the final paragraph under _"Competition"_, but after some rewriting, it's the best and most accurate I could come up with. Feedback welcome (GitHub suggestions work great for this!)

### CODE REVIEW

**Comment by mildlyincompetent** [comment]
May I ask what was misleading (even subjectively) about the original footnote? It appears to state pretty much the same thing as the paragraph which has been added - in essence that SR/RoboCon do not officially support or use the software but that we're trying to work with them to get to a point where they want to use the software?

File: README.md [hunk 1]
```diff
@@ -40,26 +40,28 @@ class Robot:
...
+We intend to support the kits of the following robotics competitions:
 
-- SourceBots
-- Student Robotics
-- Hills Road RoboCon
+- [SourceBots](https://sourcebots.co.uk/)
+- [Student Robotics](https://studentrobotics.org/)
+- [Hills Road RoboCon](https://hr-robocon.org/)
+
+Whilst `j5` isn't officially endorced by Student Robotics or RoboCon, we are working closely with Student Robotics to ensure perfect compatibility. Many `j5` contributors are members of Student Robotics and SourceBots
```
Comment (mildlyincompetent) [inline README.md]: s/endorced/endorsed

Comment (trickeydan) [inline README.md]:
```suggestion
Whilst `j5` isn't officially endorsed by Student Robotics or RoboCon, we are working closely with Student Robotics to ensure perfect compatibility. Many `j5` contributors are members of Student Robotics and SourceBots.
```

File: README.md [hunk 2]
```diff
@@ -40,26 +40,28 @@ class Robot:
...
         self.servo_board = self.servo_boards.singular()
-        
+
         self.ruggeduino = Ruggeduino(self._env)
 
 ```
 
 ## Competitions
 
-We are working with developers for the following robotics competition to support their hardware(1):
```
Comment (trickeydan) [inline README.md]: Pretty sure I'm a developer from SR? and @mildlyincompetent

Comment (mildlyincompetent) [reply README.md]: Are any of us from RoboCon? I feel that Jake's version is somewhat clearer.

Comment (trickeydan) [reply README.md]: That's true, none of are from RoboCon currently. Except Joe but he isn't really involved with `j5`

Comment (trickeydan) [reply README.md]: I'm happy for this change to go ahead to be clear.

File: README.md [hunk 3]
```diff
@@ -71,7 +73,4 @@ As this library has to support multiple hardwares, with multiple backends and st
 
 ## How do I contribute to j5?
 
-`j5` is being developed by a group of volunteers at the University of Southampton. We welcome contributions and reside in a channel on the SourceBots Slack.
```
Comment (trickeydan) [inline README.md]: At the time of writing, this was correct. However it is not anymore.

**Comment by RealOrangeOne** [comment]
The primary issue is that there's a foot note to begin with. In this context, it's much easier to just reword the section so all the information is there, rather than have part of it separately. The part which is separate (namely the fact there's no official ties (at time of writing)) is a very important part, and should go with the rest of the information.

**Pull request merged**